### PR TITLE
feat: apply data loader to CollectorScheduleForm

### PR DESCRIPTION
**Pull request by WANZARGEN**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [x] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description

Please review in order of commit.
(Commit messages say it all)

https://github.com/cloudforet-io/console/assets/26986739/97bb2f58-aea4-4e79-a612-514d6b8105b6

### CODE REVIEW

File: apps/web/src/services/asset-inventory/collector/modules/CollectorScheduleForm.vue [hunk 1]
```diff
@@ -88,6 +97,7 @@ const state = reactive({
             .get('hour')
             .toString()).sort((a, b) => Number(a) - Number(b));
     }),
+    loading: computed<boolean>(() => collectorFormState.originCollector === null),
```
Comment (yuda110) [inline apps/web/src/services/asset-inventory/collector/modules/CollectorScheduleForm.vue:97]: praise: 😮 good idea

File: apps/web/src/services/asset-inventory/collector/modules/CollectorScheduleForm.vue [hunk 2]
```diff
@@ -1,5 +1,9 @@
 <template>
-    <div class="collector-schedule-edit-form">
+    <p-data-loader :loading="state.loading"
+                   :data="true"
```
Comment (yuda110) [inline apps/web/src/services/asset-inventory/collector/modules/CollectorScheduleForm.vue:3]: question: Is this temp value?

File: apps/web/src/services/asset-inventory/collector/modules/CollectorScheduleForm.vue [hunk 3]
```diff
@@ -86,28 +85,30 @@ const hoursMatrix: string[] = range(24).map((hour) => hour.toString());
...
-    emits('update:hours', hours);
+    collectorFormStore.$patch({
+        scheduleHours: hours,
+    });
 };
 
-const handleChangeToggle = () => {
-    // TODO: change state of toggle button
+const handleChangeToggle = (value: boolean) => {
+    collectorFormStore.$patch({
```
Comment (yuda110) [inline apps/web/src/services/asset-inventory/collector/modules/CollectorScheduleForm.vue:108]: praise: 😎$patch😎

**Review by yuda110 — APPROVED** [review]
LGTM!!! nice commits

File: apps/web/src/services/asset-inventory/collector/modules/CollectorScheduleForm.vue (hunk 2 above)
Comment (WANZARGEN) [reply apps/web/src/services/asset-inventory/collector/modules/CollectorScheduleForm.vue:3]: No. this code is for hiding empty case.

File: apps/web/src/services/asset-inventory/collector/modules/CollectorScheduleForm.vue (hunk 1 above)
Comment (sulmoJ) [reply apps/web/src/services/asset-inventory/collector/modules/CollectorScheduleForm.vue:97]: > praise: 😮 good idea

I agree, too.

File: apps/web/src/services/asset-inventory/collector/modules/CollectorScheduleForm.vue (hunk 3 above)
Comment (sulmoJ) [reply apps/web/src/services/asset-inventory/collector/modules/CollectorScheduleForm.vue:108]: But.. there's no log left in the devtool..

**Review by sulmoJ — APPROVED** [review]
LGTM!!!

**Pull request merged**